bot/cogs/utils.py

```diff
@@ -128,6 +129,59 @@ def get_info(char: str) -> Tuple[str, str]:
 
         await ctx.send(embed=embed)
 
+    @staticmethod
+    def readable_time(seconds: int) -> str:
+        """Returns a number of seconds into a human-readable minutes/seconds combination."""
+        minutes, seconds = divmod(seconds, 60)
+
+        if minutes:
+            fmt = '{m}min {s}sec'
+        else:
+            fmt = '{s}sec'
+
+        return fmt.format(m=minutes, s=seconds)
```

Comment:
We have something for this in `bot/utils/time.py` but it only works for deltas. Maybe you can cast to a delta here and use that function instead?

Reply:
Done 👍